test/recipes/70-test_sslcertstatus.t

```diff
@@ -40,8 +40,6 @@ my $proxy = TLSProxy::Proxy->new(
 
 #Test 1: Sending a status_request extension in both ClientHello and
 #ServerHello but then omitting the CertificateStatus message is valid
-#TODO(TLS1.3): Temporarily disabling this test in TLS1.3 until we've completed
-#the move the status request extension to the Certificate message.
 $proxy->clientflags("-status -no_tls1_3");
```

Comment:
So we're removing the comment but leaving -no_tls1_3 in place?
That doesn't seem right.

Reply:
This test is only really relevant to TLSv1.2. It's related to the presence or absence of the CertificateStatus message which doesn't exist in TLSv1.3. The comment was added early on in the process of implementing TLSv1.3, and I think it is just stale and can be removed.